### DISCUSSION TITLE
Can we use generic pytorch LR scheduler rather than the mmengine's schedulers?

### DISCUSSION THREAD

**Discussion by hiyyg** [open-mmlab/mmengine#1370] [Q&A]
For example, we can define a very flexible lr scheduler using `torch.optim.lr_scheduler.LambdaLR`. How can we use such generic pytorch schedulers in mmengine?
